src/main/java/org/jenkinsci/plugins/workflow/steps/PwdStep.java

```diff
@@ -95,7 +97,13 @@ public boolean isTmp() {
 
         @Override protected String run() throws Exception {
             FilePath cwd = getContext().get(FilePath.class);
-            return (tmp ? WorkspaceList.tempDir(cwd) : cwd).getRemote();
+            Objects.requireNonNull(cwd);
```

Comment:
redundant

Reply:
Yes, but non in a way that is obvious from static analysis.

Reply:
If SpotBugs is complaining, you can always use `@SuppressFBWarnings` or simply

```java
assert cwd != null;
```